### DISCUSSION TITLE
Node js terminal executes infinitely

### DISCUSSION THREAD

**Discussion by SaxenaShiv** [elegantframework/elegant-cli#78] [General]
Hello,

I'm facing a problem where the terminal doesn't stop after executing the `npx elegant-init` command. As a result, the command-line interface (CLI) doesn't appear, preventing me from running subsequent commands.

Node.js Version - 18.16.0
Platform - Win 11
![image](https://github.com/elegantframework/elegant-cli/assets/74370135/02eaa0f8-3fe2-47ce-a436-521b1d0f9ddc)

**Comment by Bowens20832** [comment]
Ahhhhh! I think our node version is still behind at Node 16 still, and you are on 18 :(

On our todo list is to upgrade  Node, Next, and React all to version 18.

@Shiv-2712 Are you able to downgrade to 16 or use NVM to switch between different versions?

**Reply by SaxenaShiv** [reply]
Certainly! I possess the capability to accomplish that task.

Furthermore, it would be beneficial to include a copy icon alongside the command displayed in the documentation. This allows users to conveniently copy the command with just a single click.

**Reply by Bowens20832** [reply]
Nice feature request! I believe that could be an easy task, and very valuable to Elegant users.

**Comment by Bowens20832** [comment]
Hey @Shiv-2712,

I have been working on getting this all resolved for you this weekend.

Node, React, and Nextjs should be upgraded to their latest versions soon.

Currently in progress

![image](https://github.com/elegantframework/elegant-cli/assets/10189130/43a6ece6-de31-4870-a02c-7d7d32af7e02)

**Comment by Bowens20832** [comment]
@Shiv-2712 I am making progress on this.. I have upgraded Elegant to React 18, and all tests are passing ✅ 

I also upgraded Elegant to Node 18, by including an .npmrc file in the package that will force users to have the required node version installed on their machine.

See #84

**Reply by Bowens20832** [reply]
This issue seems to be resolved now.. I have opened a PR on the project, and published an npm prerelease to verify the fix.

```
npm install elegant-cli@innovators
```

**Reply by SaxenaShiv** [reply]
I have an Idea that in package.json it should have node dependencies and others related to it like this so that it supports 14 to all updated versions of node, you can add a `>=` sign before it for e.g: `"@types/node": ">=14.17.3",`
??
